WordPress/src/main/java/org/wordpress/android/ui/stats/StatsConnectJetpackActivity.java

```diff
@@ -6,28 +6,42 @@
 import android.support.v7.app.ActionBar;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
+import android.text.TextUtils;
 import android.view.MenuItem;
 import android.view.View;
 import android.widget.Button;
 
+import org.greenrobot.eventbus.Subscribe;
+import org.greenrobot.eventbus.ThreadMode;
 import org.wordpress.android.R;
 import org.wordpress.android.WordPress;
+import org.wordpress.android.fluxc.Dispatcher;
+import org.wordpress.android.fluxc.action.AccountAction;
+import org.wordpress.android.fluxc.generated.AccountActionBuilder;
 import org.wordpress.android.fluxc.model.SiteModel;
 import org.wordpress.android.fluxc.store.AccountStore;
+import org.wordpress.android.fluxc.store.AccountStore.OnAccountChanged;
 import org.wordpress.android.ui.JetpackConnectionWebViewActivity;
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.util.AppLog.T;
 import org.wordpress.android.util.LocaleManager;
 
 import javax.inject.Inject;
 
 import static org.wordpress.android.WordPress.SITE;
-import static org.wordpress.android.ui.JetpackConnectionWebViewActivity.Source.STATS;
+import static org.wordpress.android.ui.JetpackConnectionSource.STATS;
 
 /**
  * An activity that shows when user tries to open Stats without Jetpack connected.
  * It offers a link to the Jetpack connection flow.
  */
 public class StatsConnectJetpackActivity extends AppCompatActivity {
+    public static final String ARG_CONTINUE_JETPACK_CONNECT = "ARG_CONTINUE_JETPACK_CONNECT";
+
+    private boolean mIsJetpackConnectStarted;
```

Comment:
should we save this in `onSaveInstance`?

Reply:
I don't think so since we only want to dispatch the account fetch action or start the connection flow once.  Either the `onAccountChanged` callback will return with `mIsJetpackConnectStarted` as `false` or the connection flow will call `finish()` on `StatsConnectJetpackActivity`.

Reply:
Makes sense 👍